src/librbd/BlockGuard.h

```diff
@@ -33,7 +33,7 @@ struct BlockExtent {
 
   friend std::ostream& operator<< (std::ostream& os, const BlockExtent& block_extent) {
     os << "[block_start = " << block_extent.block_start << ", "
-       << "block_end = " << block_extent.block_end << ")";
+       << "block_end = " << block_extent.block_end << "]";
```

Comment:
I'd merge `", "` here as well and drop spaces around `=` to be consistent with most other places.

Reply:
Done. And also merged two place `", "`  in this file.